src/frontend/src/lib/components/transactions/AllTransactions.svelte

```diff
@@ -1,10 +1,23 @@
 <script lang="ts">
+	import { notEmptyString, type Token } from '@dfinity/utils';
+	import { icTransactionsStore } from '$icp/stores/ic-transactions.store';
 	import NetworksSwitcher from '$lib/components/networks/NetworksSwitcher.svelte';
 	import AllTransactionsList from '$lib/components/transactions/AllTransactionsList.svelte';
 	import MessageBox from '$lib/components/ui/MessageBox.svelte';
 	import PageTitle from '$lib/components/ui/PageTitle.svelte';
+	import { enabledNetworkTokens } from '$lib/derived/network-tokens.derived';
 	import { testnetsEnabled } from '$lib/derived/settings.derived';
 	import { i18n } from '$lib/stores/i18n.store';
+	import type { TokenUi } from '$lib/types/token';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils.js';
+
+	let enabledTokensWithoutCanister: Token[];
+	$: enabledTokensWithoutCanister = $enabledNetworkTokens.filter(
```

Comment:
@AntonioVentilii implemented a workaround last week that causes tokens with an index canister in error to fall back to using the ledger only. As a result, `transactions === null` can now have two different root causes, which are displayed differently in the UI. Can you both discuss how you'd like to handle this?

Reply:
Discussed offline with @BonomoAlessandro , and IIRC, he said that there will be two different info boxes for the tokens without index canisters and the ones with index canister not working. So it makes sense to generalize/improve the workaround I did

Did I understand correctly, @BonomoAlessandro ?

Reply:
@AntonioVentilii yess exactly.

Reply:
@peterpeterparker @AntonioVentilii 
I just added the warning message box that displays the enabled token that do not have an index canister yet. I also provided a test and updated the PR description.
The second warning message box which displays the enabled token with a broken index canister will come with antoher PR.